.github/workflows/build-and-push-head-images.yml

```diff
@@ -0,0 +1,43 @@
+name: Build and push head branch container images
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  build-and-push-oss-image:
+    runs-on: ubuntu-20.04
+    timeout-minutes: 30
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
+      - name: Set up Docker Buildx
+        id: buildx
+        uses: docker/setup-buildx-action@v1
+      - name: Login to Docker Hub
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+      - name: Create multi-arch image
```

Comment:
Just to confirm, without passing `--platform`, is it already default to multi-arch?

Reply:
Sorry, I missed tweaking this text, I just changed to a simple "create container image"